tests/unit/test_indexapi_client_multi.py

```diff
@@ -0,0 +1,67 @@
+"""
+Unit tests for indexapi
+There are more in the integration tests
+"""
+
+import pytest
```

Comment:
nothing in this file changed - i split `test_indexapi_client` into two test modules (`multi` and `single`)